### DISCUSSION TITLE
Notes for the DevPool pitch deck

### DISCUSSION THREAD

**Discussion by sergfeldman** [ubiquity/.github#22] [DevPool]
### Crypto projects pain points

**Operational limitations.**
The operational throughput is limited by the capacity of internal developers. 
The throughput scaling is not easy and smooth.

**Efficiency and total cost of development.** 
The total cost of development is pretty high, i.e. expenses on Engineering Management. 
The onboarding of new developers requires specific effort.

**Solution quality.**
The overall solution quality is limited by the level of internal developers.

### DevPool is a bounty marketplace on the GitHub for crypto projects and developers. 

**Benefits**

DevPool makes it possible to 
- Increase operational throughput and scale the development easily.
- Decrease the efficiency and total cost of development because bounty is paid only after the task is done.
- Increases overall quality because tasks are performed by the different developers that naturally audit code, find problems, and propose better solutions.

### Features

- Automation is implemented directly on GitHub which is suitable for most crypto projects.
- DevPool already contains some of the top developer talents.
- DevPool is focused on crypto, i.e. payments are in crypto which is valuable for developers.

**1) Easy scalability**
The operational throughput is significantly increased by the developers from GitHub and UviquiBot automation.
The scaling and flexibility of development are easily increased.

**2) Increased efficiency and decreased total cost of development**
The total cost of development is decreased because of the "pay only for done tasks" approach (i.e. expenses on Engineering Management).
The onboarding of new developers is simplified.

**3) GitHub-native usability**
All the GitHub tools are available.
The routine task activity is automated via UbiquiBot.
There is no need to maintain tasks on a separate freelance platform.

**4) Increased solution quality**
The overall quality of the solution increased because tasks are performed by the different developers that naturally audit code, find problems, and propose better solutions.
Fresh ideas and approaches came from the cooperation with the talents available on GitHub.

### Demo from the crypto project perspective 
1. Produce a detailed bounty specification, define bounty price and timing.
2. Answer questions and clarify requirements.
3. Review pull requests.
4. Payment will be performed automatically.

### Get started
1. Get a demo of DevPool functionality and best practices.
2. Top-up the UbiquityBot wallet for payments for the done bounties.  
3. Create the first bounties.

**Comment by sergfeldman** [comment]
### Video maker, animated video, video pitch maker 

Requirements:
- permanent free plan 
- ability to download video in 720 or higher resolution 
- basic video editing features 
- nice to have - text to speech functionality 

**Info**
https://startupstash.com/pitch-deck-tools-for-startups/
https://york.ie/blog/how-to-make-a-video-pitch-deck/
https://videoigniter.com/explainer-video-for-startup/
https://www.techradar.com/news/the-best-free-text-to-speech-software
https://www.softwaretestinghelp.com/text-to-speech-software/
https://filmora.wondershare.com/video-editing/text-to-speech-video-maker.html

**Tools**
https://wideo.co/pricing-guide/

https://simpleshow.com/plans/pricing/

https://www.canva.com/create/animated-videos/

https://invideo.io/pricing

https://fliki.ai/pricing

https://www.veed.io/create/business-video-maker/pitch-maker
https://www.veed.io/pricing

https://wave.video/pricing

https://elai.io/a-strartups-guide-to-perfect-pitching
https://elai.io/pricing

https://flixier.com/create/pitch-maker-online
https://flixier.com/pricing

https://wearenova.ai/pitch-video-maker/
https://wearenova.ai/pricing/

https://pitch.com/
https://pitch.com/pricing

https://www.flexclip.com/pricing.html

https://www.kapwing.com/pricing

**Comment by sergfeldman** [comment]
Videos 

**1. DevPool promo video for partner ptojects**

Timing 
- 2-3 minutes

Topics
- Crypto projects pain points
- How DevPool resolved pain points + brief demo
- Benefits for crypto projects
- How to start 

Style
- Business or casual, humor?
- TBD

**2. DevPool educational demo for partner projects** 

Timing
- 10-15 minutes

Topics
https://github.com/ubiquity/.github/discussions/19#discussioncomment-5192091

Style
- TBD

**Reply by 0x4007** [reply]
Well I definitely want a screencap/share video showing off a quick overview of the bot features, because it seems like it was signficantly clearer for Alpha Labs after I screenshared on the call, even after talking about it in text for some time.

**Comment by sergfeldman** [comment]
Demo from the bounty hunter perspective

- Select a bounty.
- Clarify requirements and ask questions.
- Provide a solution for the bounty.
- Get paid.

**Reply by 0x4007** [reply]
Not really a priority but it can be useful to include in the https://dao.ubq.fi/devpool-flow page

**Comment by sergfeldman** [comment]
### **Demo of the UbiquiBot automation** 

### Info 

https://github.com/ubiquity/.github/issues/35 Task with screens and video
https://github.com/ubiquity/bounty-bot/issues/220 Test task 1
https://github.com/ubiquity/bounty-bot/issues/222 Test task 2-1
https://github.com/ubiquity/bounty-bot/issues/224 Test task 2-2
https://github.com/ubiquity/.github/issues/42 Test task 3
https://docs.google.com/document/d/1v8YDkVPbxyfNNdcfKwZNCKTQL2KjR019ftNbgVRljHE/edit?usp=sharing Mocks for the demo 

Examples of technical tasks
https://github.com/ubiquity/ubiquity-dollar/issues/473 
https://github.com/ubiquity/ubiquity-dollar/issues/394 


### These points reduce spending time and keep the flow organized without human effort

**1. Setting up the bounty price**
Usable for the bounty issuer.

**2. Bounty-Developer matchmaking - comment with an invitation to take the bounty**
Suitable for the bounty issuer because it finds the most appropriate bounty hunters based on their GitHub experience.
Usable for the bounty hunter because it automatically offers him appropriate bounties.

https://github.com/ubiquity/bounty-bot/discussions/107#discussioncomment-4940811 

**3. Bot commands /help, /assign, /wallet**
Usable for both the bounty issuer and the bounty hunter.

https://github.com/ubiquity/.github/issues/35

**4. Reminders to the bounty hunter in the comments**
Usable for both the bounty issuer and the bounty hunter.

https://github.com/ubiquity/.github/issues/30 Reminder

**5. Claim the payment after done bounty and get paid**
Suitable for the bounty issuer because it does not require performing each payment manually, and also the gas fees are paid by the bounty hunter.
Usable for the bounty hunter because he can get paid at once when the bounty is accepted.

https://github.com/ubiquity/bounty-bot/issues/216#issuecomment-1488779099 Claim new

https://github.com/ubiquity/bounty-bot/discussions/68  Request Get paid
https://github.com/ubiquity/ubiquity-dollar/issues/411  Already paid

**Comment by sergfeldman** [comment]
**Pitch deck content templates** 

https://pitch.com/blog/pitch-deck-structure
https://slidebean.com/blog/pitch-deck-structure

**Slide design tools**

https://www.cyberlink.com/blog/photo-editing-best-software/1360/free-graphic-design-software

https://www.canva.com/pricing/
https://www.canva.com/solutions/startups/ 

https://www.figma.com/pricing/
https://www.figma.com/templates/
